can_tools/scrapers/official/FL/fl_vaccine.py

```diff
@@ -13,18 +16,77 @@ class FloridaCountyVaccine(StateDashboard):
     location_type = "county"
     state_fips = int(us.states.lookup("Florida").fips)
     fetch_url = "http://ww11.doh.state.fl.us/comm/_partners/covid19_report_archive/vaccine/vaccine_report_latest.pdf"
+    fetch_url_for_counties = "http://ww11.doh.state.fl.us/comm/_partners/covid19_report_archive/vaccine-county" \
+                             "/vaccine_county_report_latest.pdf"
+
     source_name = "Florida Department of Health"
 
     def fetch(self):
-        return camelot.read_pdf(self.fetch_url, pages="2-end", flavor="stream")
+        vaccine_data = camelot.read_pdf(self.fetch_url, pages="2-end", flavor="stream")
+
+        county_names = []
+        results = requests.get(self.fetch_url_for_counties)
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            tmp_file = '{}/vaccine_county_report_latest.pdf'.format(tmp_dir)
+            with open(tmp_file, 'wb') as f:
+                f.write(results.content)
+                pdf_pages_headers = textract.process(tmp_file)
+                county_names = re.findall(r"COVID-19: (?P<countyName>.*?) vaccine summary",
+                                          pdf_pages_headers.decode("utf-8"))
+                county_names = list(map(lambda x: x if not x.endswith(" County") else x[:-7], county_names))
+
+        county_demographics_data = camelot.read_pdf(self.fetch_url_for_counties, pages="1-end", flavor="stream", row_tol=10)
+        return {"vaccine_data": vaccine_data, "county_demographics_data": county_demographics_data, "county_names": county_names}
+
+    def normalize_group(self, df, demographic, dgroup, group_rename, cmu):
```

Comment:
Looks like this method is unused? You normalize each demographic data in `if` blocks in the main `normalize` method. 

I'd recommend breaking them out into individual demographic normalize methods (eg `_normalize_sex`, `_normalize_age`, etc). Then call each one in the main `normalize ` method.

I believe that's what you were starting to do with this method.